### PR TITLE
Archer - dded support for multiselect fields on the create-record and update-record commands

### PR DESCRIPTION
**Pull request by guykeller**
## Status 
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20081

## Description
Type 8 (multiselect) fields were not populated while creating on updating records.
This is now fixed.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-ArcherRSA.yml [hunk 1]
```diff
@@ -2387,7 +2387,52 @@ script:
...
+        return desiredGroupID;
+    }
+
+    function getGroupData() {
+        var groupsData = sendRestRequest("/api/core/system/group", 'archer-create-record', '', true)
+        return groupsData;
+    }
+
     // integration commands functions
+    function replaceMultiselectNameWithID(fieldsToValues, createRecordMapping) {
```
Comment (ronykoz) [inline Integrations/integration-ArcherRSA.yml]: you are not using this anywhere

File: Integrations/integration-ArcherRSA.yml [hunk 2]
```diff
@@ -2426,6 +2471,7 @@ script:
 
         if (params.useRest){
             // in case that the user wants to use the REST API
+            fieldsToValue = getIDsOfMultiselectFields(fieldsToValue, createRecordMapping);
```
Comment (ronykoz) [inline Integrations/integration-ArcherRSA.yml]: what is this function getIDsOfMultiselectFields ? can't find it in the code

File: Integrations/integration-ArcherRSA.yml (hunk 1 above)
Comment (guykeller) [reply Integrations/integration-ArcherRSA.yml]: fixed

File: Integrations/integration-ArcherRSA.yml (hunk 2 above)
Comment (guykeller) [reply Integrations/integration-ArcherRSA.yml]: fixed

**Pull request merged**